### DISCUSSION TITLE
alist+rclone 上传115异常

### DISCUSSION THREAD

**Discussion by zhongmeijun007** [AlistGo/alist#6418] [General]
alist挂载115网盘，使用默认配置通过webdav挂载到rclone，4月初开始发现单任务上传速度一开始有100+mbps,上传一会变为50-60mbps，再传一会变为30mbps，最后变为3mbps。尝试换过上传到阿里云盘，一切正常；尝试更换其他运营商的VPS、更换115账号，问题依旧。请问大家115不是托管在阿里机房么，这是115网盘单方面限速么？谢谢

**Comment by pongfcnkl** [comment]
直接上传到webdav是先上传到缓存，所以你看到的rclone速度是传输到缓存的速度也就是硬盘写入速度，和网络速度无关
另外rclone可以加timeout 0参数不会超时

**Comment by zhongmeijun007** [comment]
我看的是任务管理器的网络速度，我现在是不会超时，是VPS一直没速度，只有3Mbps

**Reply by pongfcnkl** [reply]
国外服务器吗？那挺正常的，115没有国外服务器
